Furball.Engine/Engine/DevConsole/ConsoleDrawable.cs

```diff
@@ -34,11 +37,11 @@ public class ConsoleDrawable : UiTextBoxDrawable {
             this.Selected = false;
         }
 
-        public override void Dispose(bool disposing) {
+        public override void Dispose() {
             this.OnCommit                      -= this.OnTextCommit;
-            FurballGame.InputManager.OnKeyDown -= this.OnKeyDown;
+            FurballGame.InputManager.OnKeyDown += this.OnKeyDown;
 
-            base.Dispose(disposing);
+            base.Dispose();
```

Comment:
Why are we adding to the event on dispose?

Reply:
accidentally added + reee